apps/common/behaviours/generate-pdf.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const pdf = require('html-pdf');
+const UploadModel = require('../../common/models/file-upload');
+const concat = require('concat-stream');
+const path = require('path');
+
+module.exports = superclass => class extends superclass {
+  saveValues(req, res, callback) {
+    super.saveValues(req, res, (err, values) => {
```

Comment:
I'd prefer if all of our behaviours did their thing and *then* called super. It means we have a predictable order of operations when composing behaviours.